bundle/language_en.properties

```diff
@@ -494,6 +494,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = Positive
 angal.lab.printreport                                                            = Print report
 angal.lab.printtable                                                             = Print table
+angal.lab.print																	 = Print
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d